python/cudf/cudf/core/dataframe.py

```diff
@@ -5664,12 +5664,20 @@ def cov(self, **kwargs):
         df._set_column_names_like(self)
         return df
 
-    def corr(self, method="pearson"):
+    def corr(self, method="pearson", min_periods=None):
```

Comment:
I added the `min_periods` to align with pandas. `Series.corr` has this parameter even though it is not supported, so I thought we should do the same for `DataFrame.corr`. If this is not desirable, I can revert this change and make this PR docs-only.